resource/tasks.json

```
@@ -4163,73 +4164,122 @@
             400,
             100
         ],
+        "reduceOtherTimes": [
+            "StartButton1"
+        ],
         "next": [
-            "MedicineConfirm"
+            "StartButton1",
+            "OfflineConfirm",
+            "CloseStonePage"
         ]
     },
     "MedicineConfirm": {
         "action": "ClickSelf",
-        "maxTimes": 0,
         "roi": [
             1000,
             500,
             200,
             150
         ],
-        "reduceOtherTimes": [
-            "StartButton1"
-        ],
-        "exceededNext": [
-            "ExpiringMedicineCheck",
-            "CloseStonePage"
-        ],
+        "cache": false,
+        "postDelay": 1000,
         "next": [
-            "StartButton1",
-            "OfflineConfirm",
-            "MedicineConfirmAgain@LoadingText"
+            "#self",
+            "Stop"
         ]
     },
-    "MedicineConfirmAgain": {
-        "Doc": "Refer to Task `StoneConfirmAgain`",
-        "baseTask": "MedicineConfirm",
-        "template": "MedicineConfirm.png",
-        "maxTimes": 2147483647,
-        "reduceOtherTimes": []
-    },
-    "UseStone": {
-        "action": "DoNothing",
+    "MedicineReduceIcon":{
+        "preDelay": 500,
         "roi": [
-            600,
-            130,
-            420,
-            420
+            750,
+            200,
+            500,
+            50
         ],
-        "next": [
-            "StoneConfirm"
+        "maskRange": [
+            1,
+            255
         ]
     },
-    "DrGrandetUseOriginiums": {
+    "MedicineUsingCount":{
```

```suggestion
    "UsingMedicingCount":{
```
Wouldn't this be more in-line with all the others?